cmd/soroban-rpc/internal/network/requestdurationlimiter.go

```diff
@@ -121,6 +121,9 @@ func (w *bufferedResponseWriter) WriteOut(ctx context.Context, rw http.ResponseW
 	}
 }
 
+// TODO: refactor and simplify this function
```

Comment:
should we start tracking these in github issues?

Reply:
I already created a ticket #225